### ISSUE TITLE
Publish to F-Droid

### ISSUE THREAD

**Issue by alexanderadam** [RushikeshKamewar/PrivacyDashboard#2]
Hi @RushikeshKamewar,

PrivacyDashboard is looking great and it would be nice if it could be made available via F-Droid.

It [doesn't seem to be available yet](https://search.f-droid.org/?q=privacydashboard) and I wasn't able to find [a RFP](https://gitlab.com/fdroid/rfp/-/issues?scope=all&state=opened&search=privacy+dashboard) either.

The F-Droid team [documented all necessary things](https://f-droid.org/en/docs/#developers) but I guess the only real show stopper would be non-free components (but the PrivacyDashboard components are all free, right?).

Thank you for your work! 🙏

**Comment by RushikeshKamewar** [comment]
Yes. This is my first open source app, so I'm bot really sure how to publish to F-Droid yet.

Everything in Privacy Dashboard is free without any ads.

I do accept donations in the app. I'm not sure how it'll work with F-droid as it uses google play billing library.

I could remove the donation section from the F-droid version of the app if required.

**Comment by alexanderadam** [comment]
> I'm bot really sure how to publish to F-Droid yet.

If you're fine with publishing it on F-Droid, [you could create this Request for Packaging](https://gitlab.com/fdroid/rfp/-/issues/new?issue%5Bmilestone_id%5D=).

> I'm not sure how it'll work with F-droid as it uses google play billing library.

I'm pretty sure that proprietary components like that aren't allowed.
As far as I'm aware, many apps have a different build recipe for F-Droid, where proprietary components are removed.

But I'm not part of F-Droid and I'm not an Android developer either. :wink:

**Comment by aha999** [comment]
Came here to post this issue, but i see it's already here.
Personally, if it's open source and not on F-Droid it's sort of sketchy to me, as F-droid builds from source and you can be sure that it matches the source code, it's a great system really, sot would be great if it was available there. Congrats on your first open source app! <3

**Comment by erikeah** [comment]
F-droid will be a good add and I think the public of the app will increase.

**Comment by NaNa2509** [comment]
> สวัสดี@RushikeshKamewar,
> 
> PrivacyDashboard ดูดีมาก และคงจะดีไม่น้อยหากสามารถใช้งานได้ผ่าน F-Droid
> 
> ดูเหมือนว่า จะ[ยังไม่พร้อมใช้งาน](https://search.f-droid.org/?q=privacydashboard)และฉันก็ไม่พบ[RFP](https://gitlab.com/fdroid/rfp/-/issues?scope=all&state=opened&search=privacy+dashboard)เช่นกัน
> 
> ทีม F-Droid [บันทึกทุกสิ่งที่จำเป็น](https://f-droid.org/en/docs/#developers)แต่ฉันเดาว่าตัวหยุดการแสดงที่แท้จริงเพียงตัวเดียวเท่านั้นที่จะเป็นส่วนประกอบที่ไม่ฟรี (แต่ส่วนประกอบ PrivacyDashboard นั้นฟรีทั้งหมดใช่ไหม)
> 
> ขอบคุณสำหรับการทำงานของคุณ! 🙏

Thank you👍👍👍

**Comment by NaNa2509** [comment]
ช่วยฉันแก้ปัญหาด้วย..ขอบคุณ
